tekton/tasks/binary-container-init.yaml

```diff
@@ -50,6 +52,7 @@ spec:
           cpu: 395m
       script: |
         set -x
+        set -e
```

Comment:
nitpick: this one is technically an unrelated change that should go to a separate commit. Also, can you put it on the same line as the setting above instead of running the builtin twice?

Reply:
it's related change, now task has 2 commands, thus it wouldn't work as expected because only last return status is being processed by tekton. Thus with 2 commands, I need to stop execution and return error at first occurrence.

Merged into one line